src/lib.rs

```diff
@@ -741,6 +741,7 @@
 )]
 
 pub mod backend;
+pub mod deps;
```

Comment:
I think `deps` should be `pub(crate)`, because this is internal mod.

Reply:
well, we need to expose it to use it in `tests/*`

Reply:
My point was: when we were making some internal mod public it became part of the public API  sea-query and every change is breaking...

Reply:
sure, I'll try to fix that. For now I'm trying to find a nice way to fix the CI 😉